sqlx-mysql/src/connection/mod.rs

```diff
@@ -57,6 +57,19 @@ impl MySqlConnection {
             .status_flags
             .intersects(Status::SERVER_STATUS_IN_TRANS)
     }
+
+    pub async fn nuke_cached_statements(&mut self) -> Result<(), Error> {
```

Comment:
Of course, this shouldn't just be a public method on the connection. Users might confuse it for the method that they're _supposed_ to use to clear the prepared statement cache.

The easiest route would be to just mark this `#[doc(hidden)]`.